src/webgpu/capability_info.ts

```diff
@@ -215,6 +215,17 @@ export const kTextureUsageInfo: {
 /** List of all GPUTextureUsage values. */
 export const kTextureUsages = numericKeysOf<GPUTextureUsageFlags>(kTextureUsageInfo);
 
+/** Check if `usage0` is TRANSIENT_ATTACHMENT and `usage1` is not RENDER_ATTACHMENT. */
```

Comment:
This seems like a very specific situation of 2 usages. How about 3? Or 1?

wdyt about just one usage?

```js
export function IsValidTransientAttachmentUsage(
  usage: GPUTextureUsageFlags,
): boolean {
  return (
    usage === GPUConst.TextureUsage.TRANSIENT_ATTACHMENT |
              GPUConst.TextureUsage.RENDER_ATTACHMENT
  );
}
```

And then call it with 2 usages `isValidTransientUsageAttachment(usage0 | usage1)`

Also, wdyt about making it a positive assertion?

Reply:
My understanding is that we would have to do something like that... 
```
        const usage = usage0 | usage1;
        
        // If the usage does not contain TRANSIENT, we don't need to filter it here.
        if ((usage & GPUConst.TextureUsage.TRANSIENT_ATTACHMENT) === 0) {
          return false;
        }

        // If it *does* contain TRANSIENT, exclude it unless it matches the valid combination.
        return !IsValidTransientAttachmentUsage(usage);
```

In that case, should I bundle the `usage & GPUConst.TextureUsage.TRANSIENT_ATTACHMENT` part inside `IsValidTransientAttachmentUsage` as well?